main.py

```diff
@@ -118,44 +98,46 @@ def login(self):
             stored_password = self.database.get_password(user_id)
             if stored_password == hashed_password:
                 self.logged_in_user_id = user_id
-                print(f"Login successful! Welcome back, {self.database.get_username(user_id)}.")
-            else:
-                print("Incorrect password.")
-        else:
-            print("No account found with that email.")
+                log.info(
+                    "Login successful! Welcome back, %s.", 
+                    self.database.get_username(user_id)
```

Comment:
```suggestion
                    "Login successful! Welcome back, %s."
                    % self.database.get_username(user_id)
```
isn't a `%` needed, with no `,`?

Reply:
No.

`Login successful! Welcome back, will.`

Reply:
fair enough